src/dragging/dragging.js

```diff
@@ -22,7 +22,22 @@
     let onMouseMove = function(e) {
       let deltaX = e.clientX - clientX + startX;
       let deltaY = e.clientY - clientY + startY;
-      onMove(deltaX, deltaY);
+      onMove(deltaX, deltaY, e);
+      var dragTargets = document.elementsFromPoint(e.clientX, e.clientY);
```

Comment:
Drop targets need to be picked on every move because you can move over new elements. We already do lots of hit testing during dragging, and we'll likely do more. It generally works well, and the biggest his we get is from the asynchronicity and message passing overhead of communicating with the iframe. The browser is doing a very similar thing with native drag and drop, btw.

re 2): I don't think this is possible yet, since all the handling is synchronous. It _will_ become an issue soon when we look for drop targets in the iframe. We'll have to see how bad it is and what we can do to mitigate.